orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -35,6 +36,8 @@
 from orangecanvas.gui.utils import disconnected
 from orangewidget.utils.listview import ListViewSearch
 
+from PyQt5.QtWidgets import QMessageBox
```

Comment:
Please delete this line (the correct import is already few lines up). We need to import from AnyQt so that the add-on is compatible both with Qt 5 and 6.